extensions/wikia/Email/templates/main.mustache

```diff
@@ -68,6 +68,30 @@
 						{{#footerMessages}}
 							<p class="fine-print">{{{.}}}</p>
 						{{/footerMessages}}
+
+						{{# badges.hasMobileApplicationBadges}}
+
+							<div class="spacer">
```

Comment:
You can use e.g. \<hr\> to introduce spacing between thematically different sections in html